classes/class-wc-connect-paypal-ec.php

```diff
@@ -128,19 +128,16 @@ public function maybe_init_banner( $screen ) {
 				|| ( // Edit order page.
 					'shop_order' === $screen->post_type
 					&& 'post' === $screen->base
+					&& 'ppec_paypal' === WC_Connect_Compatibility::instance()->get_payment_method( new WC_Order( $_GET['post'] ) )
```

Comment:
You can use `wc_get_order()` here with no parameters to get the current order being edited.